Added the reply button to the default tiddler html rep

bengillies opened the pull request:
I've only added the reply button to the default tiddler representation
as you can currently only reply to tiddlers. It doesn't currently make
sense to reply to collections of tiddlers, bags or recipes.

bengillies committed:
Added the reply button to the default tiddler html rep

I've only added the reply button to the default tiddler representation
as you can currently only reply to tiddlers. It doesn't currently make
sense to reply to collections of tiddlers, bags or recipes.

cdent commented on tiddlywebplugins/templates/base.html:
```diff
@@ -34,5 +34,8 @@
         <script type="text/javascript" src="{{ link }}"></script>
         {%- endfor -%}
         <script type="text/javascript" src="/bags/common/tiddlers/backstage.js"></script>
+        <script type="text/javascript" src="/bags/common/tiddlers/jquery.js"></script>
```
If' we're not using jquery on the other reps yet, should this change not be in tiddler.html instead?

I kind of assume that jquery will be needed everywhere eventually, but that's eventually.

bengillies replied:
yep, good spot.

bengillies committed:
Moved jQuery into tiddler.html

As spotted by @cdent. As jQuery is only used by the tiddler repr it only
makes sense to include it there.
